src/main/java/com/hedera/hashgraph/sdk/crypto/Mnemonic.java

```diff
@@ -35,18 +37,68 @@
 
     private static final SecureRandom secureRandom = new SecureRandom();
 
-    public Mnemonic(List<CharSequence> wordList) {
-        if (wordList.size() != 24) {
-            throw new IllegalArgumentException("wordList must have length 24");
+    private static final Set<CharSequence> wordList;
+
+    static {
+        // the BIP39 lib doesn't give us direct access to the word list
+        HashSet<String> newWordList = new HashSet<>(2048);
+
+        for (int i = 0; i < 2048; i++) {
+            newWordList.add(English.INSTANCE.getWord(i));
+        }
+
+        wordList = Collections.unmodifiableSet(newWordList);
+    }
+
+    /**
+     * Construct a mnemonic from a 24-word list.
+     *
+     * @throws BadMnemonicError if {@code words.size() != 24} or if any words in the list do not
+     * exist in the BIP-39 standard English word list (as typo detection).
+     *
+     * @param words the 24-word list that constitutes a mnemonic phrase.
+     */
+    public Mnemonic(List<? extends CharSequence> words) {
```

Comment:
I'm decently sure this is a forward-compatible change (`List<CharSequence>` -> `List<? extends CharSequence>`) but it makes the constructor much nicer to use, e.g. if you have a `List<String>` (which won't automatically coerce to `List<CharSequence>`).